client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-hosting-details-list.tsx

```diff
@@ -25,9 +28,34 @@ export const useUpgradePlanHostingDetailsList = () => {
 			),
 			icon: trendingUp,
 		};
-		hostingDetails.splice( 1, 1 );
-		hostingDetails.unshift( updatedHostingSpeedDetails );
+		newDefaultHostingDetails = newDefaultHostingDetails.filter(
+			( { id } ) => id !== 'increased-speed'
+		);
+		newHostingDetails.push( updatedHostingSpeedDetails );
+	}
+
+	// FID hosting details.
+	if ( fidPercentageDifference > 0 ) {
+		const wordpressFID = Math.round( 100 * wordpressCwvtechReportJson?.goodFID );
+
+		const updatedHostingFasterResponseDetails = {
+			title: translate( 'Faster response' ),
+			description: translate(
+				'%(wordpressFidPercentage)s of sites on WordPress.com respond at least %(sitePercentageDifference)s faster than yours on the first interaction.',
+				{
+					args: {
+						wordpressFidPercentage: `${ wordpressFID }%`,
+						sitePercentageDifference: `${ fidPercentageDifference }%`,
+					},
+				}
+			),
+			icon: next,
+		};
+		newDefaultHostingDetails = newDefaultHostingDetails.filter(
+			( { id } ) => id !== 'reduced-error-rate'
```

Comment:
When we add the "Faster response", I'm removing the "Reduced error rate". WDYT? If I understand well, we always want to remove one when we add another item to not grow the list too much.